### DISCUSSION TITLE
Meaning of conservative variance estimate

### DISCUSSION THREAD

**Discussion by mkenglish** [FrederickHuangLin/ANCOMBC#68] [Q&A] [answered]
I anticipated a high number of differentially abundant taxa in the dataset that I used `ancombc` on, so I set `conserve = FALSE`, as the manual says to do. Where exactly does this parameter come into play? In the tests that `ancombc` runs under the hood, what does a conservative variance estimate mean? 
Thank you for any and all help!

**Comment by FrederickHuangLin** [comment, marked as answer]
Hi @mkenglish,

Thanks for your inquiry! The `conserve` argument specifies whether we would like to take into account the variance of the bias term in calculating the total variance of the estimated differential abundance. 

Specifically, when `conserve = FALSE`, we are using equation (33) in the [paper](https://www.nature.com/articles/s41467-020-17041-7#Sec7); while when `conserve = TRUE`, we are using equation (35) in the [paper](https://www.nature.com/articles/s41467-020-17041-7#Sec7).

Best,
Huang

**Reply by mkenglish** [reply]
Thank you so much!
